frontend/pages/policies/PolicyPage/components/PolicyResults/PolicyResults.tsx

```diff
@@ -136,9 +136,27 @@ const QueryResults = ({
     );
   };
 
-  const renderTable = () => {
+  const renderPassFailPcts = () => {
+    const { yes: yesCt, no: noCt } = getYesNoCounts(hostResponses);
+    return (
+      <span className={`${baseClass}__results-pass-fail-pct`}>
+        {" "}
+        (Yes:{" "}
+        <TooltipWrapper tipContent={`${yesCt} host${yesCt !== 1 ? "s" : ""}`}>
+          {Math.ceil((yesCt / hostsCount.successful) * 100)}%
+        </TooltipWrapper>
+        , No:{" "}
+        <TooltipWrapper tipContent={`${noCt} host${noCt !== 1 ? "s" : ""}`}>
+          {Math.floor((noCt / hostsCount.total) * 100)}%
```

Comment:
should this be `/ hostsCount.successful` instead of `/ hostsCount.total`?

Reply:
I'm getting funky numbers that don't add to 100% I believe because of the different denominators:

![Uploading Screenshot 2024-04-16 at 11.54.37 AM.png…]()